Doc/whatsnew/3.7.rst

```diff
@@ -1214,7 +2137,7 @@ Changes in the Python API
   (Contributed by Serhiy Storchaka in :issue:`29192`.)
 
 * ``Module``, ``FunctionDef``, ``AsyncFunctionDef``, and
```

Comment:
I suggest to move this entry to the beginning of the sections, as it breaks several third-party projects.